rest/tests/test_vocabulary.py

```diff
@@ -45,6 +45,22 @@ def test_vocabularies(self):
 
         self.create_vocabulary(self.repo.slug)
         vocabularies = self.get_vocabularies(self.repo.slug)
+
+        # assert that vocabularies are sorted.
+        try:
+            # for python 3
+            xrange
+        except NameError:
+            xrange = range
+
+        self.assertTrue(
+            all(
+                vocabularies['results'][i]['name'] <=
+                vocabularies['results'][i+1]['name']
+                for i in xrange(len(vocabularies['results']) - 1)
+            )
+        )
+
```

Comment:
Is this test actually necessary, given that we're comparing the return from the api with a sorted list in https://github.com/mitodl/lore/blob/9a9ce66453abdf46bb7fc24bfc13bb9469441239/rest/tests/test_vocabulary.py#L512-L521 ?